datagateway_api/src/resources/search_api_endpoints.py

```diff
@@ -31,7 +34,27 @@ class Endpoint(Resource):
         def get(self):
             filters = get_filters_from_query_string("search_api", entity_name)
             log.debug("Filters: %s", filters)
-            return get_search(entity_name, filters), 200
+            log.debug("entity_name: %s", entity_name)
+
+            if not is_query_parameter_enabled(filters):
+                entities = get_search(entity_name, filters)
+                return entities, 200
+            else:
+                query = get_search_api_query_filter_list(filters)[0].value
+
+                entities = get_search(
+                    entity_name,
+                    filters,
+                    "LOWER(o.summary) like '%" + query.lower() + "%'",
+                )
+                log.debug(
+                    "Applying score to %s entities with query %s",
+                    len(entities),
+                    query,
+                )
+                log.debug("Entities retrieved %s", len(entities))
```

Comment:
`len(entities)` is logged in the log statement above, so this one could be removed